Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -323,64 +323,50 @@ static void print(string toPrint)
             }
 
             //-------------------------------------------------------------------------
-            public void AddConnectLoginListener(IEOSOnConnectLogin connectLogin)
+            public void AddConnectLoginListener(IConnectInterfaceEventListener connectLogin)
             {
                 s_onConnectLoginCallbacks.Add(connectLogin.OnConnectLogin);
             }
-
-            public void AddAuthLoginListener(IEOSOnAuthLogin authLogin)
+            public void RemoveConnectLoginListener(IConnectInterfaceEventListener connectLogin)
             {
-                s_onAuthLoginCallbacks.Add(authLogin.OnAuthLogin);
-            }
-
-            public void AddAuthLogoutListener(IEOSOnAuthLogout authLogout)
-            {
-                s_onAuthLogoutCallbacks.Add(authLogout.OnAuthLogout);
+                s_onConnectLoginCallbacks.Remove(connectLogin.OnConnectLogin);
             }
 
-            public void AddApplicationCloseListener(Action listener)
+            public void AddAuthInterfaceEventListener(IAuthInterfaceEventListener listener)
             {
-                s_onApplicationShutdownCallbacks.Add(listener);
+                s_onAuthLoginCallbacks.Add(listener.OnAuthLogin);
+                s_onAuthLogoutCallbacks.Add(listener.OnAuthLogout);
             }
 
-            public void RemoveConnectLoginListener(IEOSOnConnectLogin connectLogin)
+            public void RemoveAuthInterfaceEventListener(IAuthInterfaceEventListener listener)
             {
-                s_onConnectLoginCallbacks.Remove(connectLogin.OnConnectLogin);
+                s_onAuthLoginCallbacks.Remove(listener.OnAuthLogin);
+                s_onAuthLogoutCallbacks.Remove(listener.OnAuthLogout);
             }
 
-            public void RemoveAuthLoginListener(IEOSOnAuthLogin authLogin)
-            {
-                s_onAuthLoginCallbacks.Remove(authLogin.OnAuthLogin);
-            }
-
-            public void RemoveAuthLogoutListener(IEOSOnAuthLogout authLogout)
+            public void AddApplicationCloseListener(Action listener)
             {
-                s_onAuthLogoutCallbacks.Remove(authLogout.OnAuthLogout);
+                s_onApplicationShutdownCallbacks.Add(listener);
             }
 
             //-------------------------------------------------------------------------
             public T GetOrCreateManager<T>() where T : IEOSSubManager, new()
             {
-                T manager = default;
                 Type type = typeof(T);
+                T manager;
                 if (!s_subManagers.ContainsKey(type))
                 {
                     manager = new T();
                     s_subManagers.Add(type, manager);
 
-                    if (manager is IEOSOnConnectLogin)
+                    if (manager is IConnectInterfaceEventListener connectListener)
```

Comment:
These also don't need to change.

Reply:
They have been restored to the original implementation.